.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
npm can't be used to develop the repo since it uses yarn workspaces, is this still file relevant for yarn?